pkg/testframework/console-operator.go

```diff
@@ -77,6 +77,11 @@ func ManageConsole(t *testing.T, client *Clientset) error {
 		return fmt.Errorf("unable to change console operator state to 'Managed': %s", err)
 	}
 
+	err = AreConsoleResourcesAvailable(client)
```

Comment:
Is this the change that catches the flake?

Reply:
Yes ! Should be :)

Reply:
awesome.